core/src/main/scala/org/http4s/HttpRoutes.scala

```diff
@@ -59,9 +58,9 @@ object HttpRoutes {
     * @return An [[HttpRoutes]] that returns some [[Response]] in an `OptionT[F, ?]`
     * wherever `pf` is defined, an `OptionT.none` wherever it is not
     */
-  def of[F[_]](pf: PartialFunction[Request[F], F[Response[F]]])(
-      implicit F: Sync[F]): HttpRoutes[F] =
-    Kleisli(req => OptionT(F.suspend(pf.lift(req).sequence)))
+  def of[F[_]: Applicative](pf: PartialFunction[Request[F], F[Response[F]]])(
```

Comment:
I usually don't mix context bounds and implicit parameters.  Once I name one, I name them all.  I don't know how others feel.

Reply:
I don't mind mixing, but obviously ok with whatever